src/ui/components/Game/useGame.test.ts

```diff
@@ -38,10 +38,12 @@ describe('useGame', () => {
     // Setup mocks
     vi.mocked(ActorContext.useActorRef).mockReturnValue(mockActorRef)
     vi.mocked(useGameRules).mockReturnValue({
-      eventCardsThatCanBePlayed: 1,
-      game: mockGame,
+      game: {
+        ...mockGame,
+        selectedWaterCardInHandIdx: 0,
+        winner: null,
+      },
       gameState: GameState.UNINITIALIZED,
-      selectedWaterCardInHandIdx: 0,
       winner: null,
```

Comment:
It seems that `winner` should only be within `game` here, so this line can probably be removed.